sentry_sdk/integrations/redis/utils.py

```diff
@@ -44,38 +44,53 @@ def _get_safe_command(name, args):
     return command
 
 
+def _key_as_string(key):
+    # type: (Any) -> str
+    if isinstance(key, dict):
+        key = ", ".join(x.decode() if isinstance(x, bytes) else x for x in key.keys())
+    elif isinstance(key, (list, tuple)):
+        key = ", ".join(x.decode() if isinstance(x, bytes) else x for x in key)
```

Comment:
We can simplify this

```suggestion
    if isinstance(key, (dict, list, tuple)):
        key = ", ".join(x.decode() if isinstance(x, bytes) else x for x in key)
```